### DISCUSSION TITLE
Disable a motd

### DISCUSSION THREAD

**Discussion by arc9531** [TownyAdvanced/Towny#4917] [Q&A] [answered]
Whenever someone logs onto the server it shows "[Towny] Modes Set: General" how do I disable that?

**Comment by LlmDl** [comment, marked as answer]
From the Towny chatconfig.yml:

```
# If true players will see [Towny] Modes set: general when they log in.
display_modes_set_on_join: 'true'
```

Set the above to false and use `/townychat reload`.
